packages/react-magma-dom/src/components/Datagrid/Datagrid.tsx

```diff
@@ -138,6 +141,11 @@ export interface UncontrolledSelectedRowsProps {
   defaultSelectedRows?: (string | number)[];
 }
 
+export const StyledCaption = styled.caption`
+  display: flex;
+  flex: 1;
+`;
```

Comment:
We should check with @orion-cengage what the default heading title styles should be

Reply:
Since you're already exporting it, we should define the StyledCaption in only one place and use it in both Table and Datagrid so they will always be consistent.